homeassistant/components/ccl/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""Config flow for CCL Electronics."""
+
+from __future__ import annotations
+
+import secrets
+from typing import Any
+
+from yarl import URL
+
+from homeassistant.components import webhook
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_WEBHOOK_ID
+from homeassistant.helpers.network import get_url
+
+from .const import DOMAIN
+
+
+class CCLConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow."""
+
+    _webhook_id: str
+    _host: str
+    _port: str
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle the initial step."""
+
+        if user_input is None:
```

Comment:
```suggestion
        if user_input is not None:
```
please invert the flow, to be more in standard with other integrations